eng/pipelines/handlers.yml

```diff
@@ -75,14 +75,14 @@ parameters:
         vmImage: $(windowsNet6VmImage)
         bootsAndroid: $(Android.Msi)
         bootsiOS: $(iOS.Msi)
-        artifact: net6-windows
+        artifact: net7-windows
```

Comment:
Probably should just remove version numbers here.

Reply:
I'm going to put `build-windows` and `build-macos`, I think.